modules/tensor_mechanics/include/materials/HyperbolicViscoplasticityStressUpdate.h

```diff
@@ -32,10 +32,10 @@ class HyperbolicViscoplasticityStressUpdate : public RadialReturnStressUpdate
 protected:
   virtual void initQpStatefulProperties() override;
 
-  virtual void computeStressInitialize(Real effectiveTrialStress,
-                                       const RankFourTensor & elasticity_tensor) override;
-  virtual Real computeResidual(Real effectiveTrialStress, Real scalar) override;
-  virtual Real computeDerivative(Real effectiveTrialStress, Real scalar) override;
+  virtual void computeStressInitialize(Real effective_trial_stress,
+                                       const RankFourTensor & /*elasticity_tensor*/) override;
+  virtual Real computeResidual(const Real effective_trial_stress, const Real scalar) override;
```

Comment:
In ```computeResidual``` and ```computeDerivative``` you don't really need a ```const``` for the Real parameters.  I'm guessing you want to make the inner workings of your code really obvious to the reader.  But if so, perhaps you should also include a ```const``` in ``computeStressInitialize``` too?